composables/useSingleReport.js

```diff
@@ -46,7 +46,7 @@ export const useSingleReport = () => {
       $fetch(`/api/reports/${route.params.id}`, {
         method: "PUT",
         body: {
-          ...report.value,
+          ...report.value.state,
```

Comment:
The difference is because we used to use the packageState (https://github.com/axodotdev/harbor/pull/38/files#diff-73c4038e65f57f315724f998e9e12b6a85fa911c104b03824aacab85b2e1f010L11) and that didn't save everything but only the `report.state` and now we just have everything in one place